### ISSUE TITLE
DetailsList: [Accessibility][Screen Reader]: Screen reader announces "group" for column header.

### ISSUE THREAD

**Issue by AbhijitUI** [microsoft/fluentui#13779]
### Environment Information

Environment:
OS: Win 10 (2004)
Build: OS Build (19613.1005)
Browser: Edge Chromium DEV Build (84.0.522.5) , Firefox(76.0)
ScreenReader: Narrator, NVDA

### Describe the issue:

ScreenReader announces group while navigating on Table column header. It announces it as "group column header name header row 1 of 2".

### Please provide a reproduction of the issue in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist
This is to notify you that the issue can be replicated on DetailsList component from Office UI Fabric, we are able to reproduce the same by exporting the same to Codepen.

**Replicable in all the CodePen's for DetailsList component**

#### Actual behavior:

ScreenReader announces group while navigating on Table column header. It announces it as "group column header name header row 1 of 2".

#### Expected behavior:

ScreenReader should not announce group while navigating on Table column header. It should announce it as "Name column header 1 of 2".

#### Priorities and help requested (not applicable if asking question):
Are you willing to submit a PR to fix? - No

Requested priority: (Blocking, High, Normal, Low) - Normal

Products/sites affected: (if applicable) - N/A

**Comment by paulgildea** [comment]
@xugao Let's determine the impact of this issue to see if this meets the bar for fixing in the DetailsList

**Comment by xugao** [comment]
@AbhijitUI - the codepen you had provided is empty. could you check and fix?

**Comment by AbhijitUI** [comment]
@paulgildea @xugao Thanks for taking a look into this issue.

@xugao This is to notify you that the issue can be replicated on DetailsList component from Office UI Fabric, https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist, we are able to reproduce the same by exporting the same to Codepen, Thanks!

**Comment by paulgildea** [comment]
@AbhijitUI Thanks for the repro we really appreciate it.

**Comment by SalmanSaberi** [comment]
Hi Team
Please let us gives us an ETA by when this issue can be closed ?
